R/deprecated.R

```diff
@@ -76,15 +69,22 @@ add_p_ <- function(...) {
 }
 
 # v1.3.3 -----------------------------------------------------------------------
-as_flextable.gtsummary <- function(x, include = everything(), return_calls = FALSE,
-                                   strip_md_bold = TRUE, group_header = NULL, ...) {
-  # soft deprecated on 2020-07-22 ----------------------------------------------
-  lifecycle::deprecate_warn("1.3.3",
-                            "gtsummary::as_flextable.gtsummary()",
-                            "as_flex_table()")
+as_flextable <- function(...) {
```

Comment:
Oh my! Very good catch.  I forgot to add the `@export` tag for the old `as_flextable()`! 🤦 

``` r
library(flextable)
library(gtsummary)
#> 
#> Attaching package: 'gtsummary'
#> The following object is masked from 'package:flextable':
#> 
#>     as_flextable

trial %>%
  tbl_summary(by = trt) %>%
  as_flextable()
#> Warning: `as_flextable()` is deprecated as of gtsummary 1.3.3.
#> Please use `as_flex_table()` instead.
#> The `as_flextable()` function graduated from 'Experimental' status in
#> v1.3.3. The function's name was changed to avoid a name conflict with
#> `flextable::as_flextable()`. If you are trying to use the function from
#> {flextable}, for the time being, use the double colon notation when both
#> {gtsummary} and {flextable} are loaded, e.g. `flextable::as_flextable(...)`.
#> This warning is displayed once every 8 hours.
#> Call `lifecycle::last_warnings()` to see where this warning was generated.
#> Error in knit_print.flextable(x, ...): unsupported format for flextable rendering:markdown_strict
```

<sup>Created on 2020-08-01 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

@karissawhiting